compiler/patterns.nim

```diff
@@ -77,7 +77,7 @@ proc checkTypes(c: PPatternContext, p: PSym, n: PNode): bool =
   if isNil(n.typ):
     result = p.typ.kind in {tyVoid, tyStmt}
   else:
-    result = sigmatch.argtypeMatches(c.c, p.typ, n.typ)
+    result = sigmatch.argtypeMatches(c.c, p.typ, n.typ, from_hlo = true)
```

Comment:
Nitpick: It should be named ``fromHlo``, the Nim compiler does not use underscores.

Reply:
np, changed to camel case